### PR TITLE
update consent and support yearly pat query for facility contact enc

### PR DESCRIPTION
**Pull request by juliangruendner**

### CODE REVIEW

**Comment by juliangruendner** [comment]
@tobiasgirardet  - added option for yearly query via type einrichtungskontakt

**Comment by juliangruendner** [comment]
@tobiasgirardet  maybe you can also have a look at the queries for pat per year - i used the type and not the profile 
@tobiasgirardet , @alexzautke  please confirm that this is correct

**Review by alexzautke — APPROVED** [review]

File: site-client/aktin-fhir-mii-report-client.py [hunk 1]
```diff
@@ -30,6 +30,8 @@
                     help='https proxy url for your fhir server - None if not set here', nargs="?", default=None)
 parser.add_argument(
     '--sendreport', help='https proxy url for your fhir server - None if not set here', action='store_true', default=False)
+parser.add_argument(
+    '--patyearfacility', help='https proxy url for your fhir server - None if not set here', action='store_true', default=False)
```
Comment (tobiasgirardet) [inline site-client/aktin-fhir-mii-report-client.py]: There is a copy-and-paste-error in the description.

File: site-client/aktin-fhir-mii-report-client.py [hunk 2]
```diff
@@ -193,6 +196,10 @@ def execute_pat_year_queries():
     while cur_year <= last_year:
         pat_ids = set()
         query = f'/Encounter?date={str(cur_year)}&_count=500'
+
+        if pat_year_with_facility:
+            query = f'/Encounter?type=http://fhir.de/CodeSystem/Kontaktebene|einrichtungskontakt&date={str(cur_year)}&_count=500'
```
Comment (tobiasgirardet) [inline site-client/aktin-fhir-mii-report-client.py:204]: The query looks fine for me and is working as expected on our server. But I am not sure if | needs to be escaped here.

**Review by tobiasgirardet — APPROVED** [review]
Minor findings.

File: site-client/aktin-fhir-mii-report-client.py (hunk 2 above)
Comment (tobiasgirardet) [reply site-client/aktin-fhir-mii-report-client.py:204]: Escaping is not needed here.

**Pull request merged**